.github/workflows/model-badge.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright DB Netz AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: generate model badge
+
+on: push
+
+jobs:
+  generate-model-badge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.11
+      - name: Generate model badge
+        run: |
+          git fetch
+          git reset
+          pip install "git+https://github.com/DSD-DBS/py-capellambse.git@$CAPELLAMBSE_REVISION"
```

Comment:
is there a reason why we don't do `pip install capellambse`?

Reply:
It's a bit more flexible, because the `CAPELLAMBSE_REVISION` can be specified. This is helpful when `py-capellambse` introduced a fix or new feature, which we want to use before it is merged.

Reply:
alright I see - shall we introduce that also in the workflow of the diagram cache?

Reply:
Sure, feel free to do so :) 
Whenever something makes sense, just do it 💪